<?xml version="1.0" encoding="UTF-8"?>
<!--
 ~ Copyright (c) 2021 the QUsb2Snes authors.
 ~
 ~ This file is part of QUsb2Snes.
 ~ (see https://github.com/Skarsnik/QUsb2snes).
 ~
 ~ QUsb2Snes is free software: you can redistribute it and/or modify
 ~ it under the terms of the GNU General Public License as published by
 ~ the Free Software Foundation, either version 3 of the License, or
 ~ (at your option) any later version.
 ~
 ~ QUsb2Snes is distributed in the hope that it will be useful,
 ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
 ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 ~ GNU General Public License for more details.
 ~
 ~ You should have received a copy of the GNU General Public License
 ~ along with QUsb2Snes.  If not, see <https://www.gnu.org/licenses/>.
 -->

<ui version="4.0">
 <class>SNESClassicMemoryFinder</class>
 <widget class="QMainWindow" name="SNESClassicMemoryFinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SNESClassicMemoryFinder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nothing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>551</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Follow these instructions first :&lt;/p&gt;&lt;p&gt;- Mod your SNES Classic with Hakchi2CE &lt;a href=&quot;https://github.com/TeamShinkansen/Hakchi2-CE/releases&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://github.com/TeamShinkansen/Hakchi2-CE/releases&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;- Install the server-stuff mod provided with QUsb2Snes&lt;br/&gt;copy the serverstuff.hmod to the user_mods directory of Hakchi2&lt;br/&gt;then in the Hakchi2 install the mod&lt;/p&gt;&lt;p&gt;- Download the &lt;a href=&quot; http://www.nyo.fr/~skarsnik/snesstuff/usb2snes-testlorom.sfc&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;usb2snes-testlorom&lt;/span&gt;&lt;/a&gt; and install it on your SNES classic&lt;/p&gt;&lt;p&gt;- Start the rom (it should show an orange screen) then hit the scan memory button&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="scanButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
